packages/api/app/Jobs/SendMail.js

```diff
@@ -0,0 +1,24 @@
+const Mail = use('Mail');
+const Config = use('Config');
+
+class UserRegisterEmail {
```

Comment:
If I'm not mistaken, this job is gonna be used to send all kinds of emails, right? If so, it'd be cool to change this name to something more generic 👍

Reply:
I copied and pasted from another job and forgot to rename the class hahaha 😅